docs/user/InstructionsForAutoTuner.md

```diff
@@ -0,0 +1,201 @@
+# Instructions for AutoTuner with Ray
+
+_AutoTuner_ is a “no-human-in-loop” parameter tuning framework for commercial and academic RTL-to-GDS flows. 
+AutoTuner provides a generic interface where users can define parameter configuration as JSON objects. 
+This enables AutoTuner to easily support various tools and flows. AutoTuner also utilizes [METRICS2.1](https://github.com/ieee-ceda-datc/datc-rdf-Metrics4ML) to capture PPA 
+of individual search trials. With the abundant features of METRICS2.1, users can explore various reward functions that steer the flow autotuning to different PPA goals. 
+
+AutoTuner provides two main functionalities as follows.
+* Automatic hyperparameter tuning framework for OpenROAD-flow-script (ORFS)
+* Parametric sweeping experiments for ORFS
+
+
+AutoTuner contains top-level Python script for ORFS, each of which implements a different search algorithm. Current supported search algorithms are as follows.
+* Random/Grid Search 
+* Population Based Training ([PBT](https://deepmind.com/blog/article/population-based-training-neural-networks))
+* Tree Parzen Estimator ([HyperOpt](http://hyperopt.github.io/hyperopt))
+* Bayesian + Multi-Armed Bandit ([AxSearch](https://ax.dev/))
+* Tree Parzen Estimator + Covariance Matrix Adaptation Evolution Strategy ([Optuna](https://optuna.org/))
+* Evolutionary Algorithm ([Nevergrad](https://github.com/facebookresearch/nevergrad))
+
+User-settable coefficient values (`coeff_perform`, `coeff_power`, `coeff_area`) of three objectives to set the direction of tuning are written in the script. Each coefficient is expressed as a global variable at the `get_ppa` function in `PPAImprov` class in the script (`coeff_perform`, `coeff_power`, `coeff_area`). Efforts to optimize each of the objectives are proportional to the specified coefficients.
+
+
+## Input JSON structure
+
+Sample JSON file for sky130hd aes design: [[link]](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/flow/designs/sky130hd/aes/autotuner.json)
+
+Simple Example with comments:
+```json
+{
+    "_SDC_FILE_PATH": "constraint.sdc",     // pointer of the base SDC file for modification
+    "_SDC_CLK_PERIOD": {                    // parameter name (clock period) for sweeping/tuning. 
+        "type": "float",                    // parameter type for sweeping/tuning
```

Comment:
JSON does not support comments. If someone tries to copy this code will generate errors. I would prefer to describe the format in a paragraph.